src/os/os2/os2.cpp

```diff
@@ -154,27 +154,25 @@ void ShowOSErrorBox(const char *buf, bool system)
 	WinTerminate(hab);
 }
 
-bool GetClipboardContents(char *buffer, const char *last)
+std::optional<std::string> GetClipboardContents()
 {
 /* XXX -- Currently no clipboard support implemented with GCC */
 #ifndef __INNOTEK_LIBC__
 	HAB hab = 0;
 
-	if (WinOpenClipbrd(hab))
-	{
-		const char *text = (const char*)WinQueryClipbrdData(hab, CF_TEXT);
+	if (WinOpenClipbrd(hab)) {
+		const char *text = (const char *)WinQueryClipbrdData(hab, CF_TEXT);
```

Comment:
`static_cast` if you are changing it anyway? Or is it a `const_cast`? Can't tell :D

Reply:
According to the specification it returns a `ULONG`. No idea whether that can be `static_cast` or `const_cast`. So, I rather leave it as is and just fix the coding style mistakes.

Reply:
You are lucky this is code that hasn't been compiled in months / years .. @orudge : you had to intention to maintain OS/2 port, but is that still realistic? I don't mind etc, but as we also don't have a CI, I am starting to worry more and more if it actually still works :)